### ISSUE TITLE
a11y: Color Contrast - Import Extension Catalog

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13712]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #3D98D3
Background Text Color: #FFFFFF
Ratio: 3.16:1 
Main Content
- Catalog Image Reference input field (on hover and on focus)
- Image Pull Secrets dropdown (on hover and on focus)
- Cancel Button (on default state) 

Foreground Text Color: #FFFFFF
Background Text Color: #3D98D3
Ratio: 3.16:1 
Main Content
- Load Button (on default state)

Foreground Text Color: #FFFFFF
Background Text Color: #297DB4
Ratio: 4.48:1 
Main Content
- Cancel and Load Button (on focus)  

Foreground Text Color: #BEC1D2
Background Text Color: #FFFFFF
Ratio: 1.78:1 
Main Content
- e.g. hub.docker.io/example-org/my-image:latest

Foreground Text Color: #FFFFFF
Background Text Color: #F64747
Ratio: 3.55:1
Main Content
- Error Button (available after adding invalid values in the input field) 
- Unable to determine image name error message (available after adding invalid values in the input field) 

"

## Expected results

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
> 
> Instances:
> Foreground Text Color: #BEC1D2
> Background Text Color: #FFFFFF
> Ratio: 1.78:1 
> Main Content
> - e.g. hub.docker.io/example-org/my-image:latest
>
> Foreground Text Color: #2F68E3
> Background Text Color: #EDF2FC
> Ratio: 4.4:1
> Main Content
> - Cancel Button (on focus and hover)  
>
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text."

Issue 224
